deals/deals_test.go

```diff
@@ -25,22 +24,22 @@ func TestMain(m *testing.M) {
 }
 
 func TestStore(t *testing.T) {
-	numMiners := []int{1, 3}
+	numMiners := []int{1}
```

Comment:
Make tests lighter to free GH CI... the custom runner was also having problems, so better use the free one.
I still have pending looking for another solution to this... in my personal laptop heavier tests run correctly.